presto-testing/src/main/java/io/prestosql/testing/AbstractTestIntegrationSmokeTest.java

```diff
@@ -441,18 +442,20 @@ public void testSelectInformationSchemaColumns()
                         "('views')");
     }
 
-    protected void assertPushedDown(String sql)
+    protected void assertPushedDown(@Language("SQL") String sql)
     {
-        assertPushedDown(sql, sql);
-    }
+        String catalog = getSession().getCatalog().orElseThrow();
+        Session withoutPushdown = Session.builder(getSession())
+                .setCatalogSessionProperty(catalog, "allow_aggregation_pushdown", "false")
+                .build();
 
-    protected void assertPushedDown(String actual, String expectedOnH2)
-    {
-        assertQuery(actual, expectedOnH2);
+        MaterializedResult actualResults = computeActual(sql);
+        MaterializedResult expectedResults = computeActual(withoutPushdown, sql);
```

Comment:
Please capture the plan for the expected `wihoutPushdown` query and assert that it has different shape than plan for actual query (with pushdown).